docs/en/latest/getting-started.md

```diff
@@ -216,7 +216,7 @@ $ curl http://127.0.0.1:9080/apisix/admin/consumers -H 'X-API-KEY: edd1c9f034335
 Next, let's bind our `Consumer(John)` to that `Route`, we only need to **Enable** the [key-auth](./plugins/key-auth.md) plugin for that `Route`:
 
 ```bash
-curl http://127.0.0.1:9080/apisix/admin/routes/5 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+$ curl http://127.0.0.1:9080/apisix/admin/routes/5 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```

Comment:
Actually I think it would be a good idea to remove all the `$`, so users could directly use the copy button on the website to get the 'runnable' command. cc @juzhiyuan 

Another reason could be, the dollar sign is often used when trying to separate commands and outputs. Since in this doc, we only list the command, so there seems no need to use `$`

Reply:
it's reasonable though I would prefer adding `$` 😂 what do you think? @spacewander

Reply:
Found some examples:
- Kubernetes: do not have `$` (https://kubernetes.io/docs/tutorials/hello-minikube/)
- Envoy: have `$` but the copy button does not fetch it (https://www.envoyproxy.io/docs/envoy/latest/start/quick-start/run-envoy)
- Istio: same with envoy (https://istio.io/latest/docs/setup/getting-started/)

So it seems the copy button could be customized

Reply:
😂 wow, that button could omit `$`

Reply:
We can omit the `$`. `$` is not part of the bash script.